Pedersen_N02/AmrWind/abl.inp

```diff
@@ -77,3 +77,81 @@ diffusion.mg_rtol = 1.0e-6
 diffusion.mg_atol = 1.0e-12
 temperature_diffusion.mg_rtol = 1.0e-10
 temperature_diffusion.mg_atol = 1.0e-13
+
+# Activate data probe sampling
+incflo.post_processing = sampling
+
+# Frequency of output for the data
+sampling.output_frequency = 5
+# Type of probes to output
+sampling.labels = l_v11 l_v12 l_v13 l_v21 l_v22 l_v23 l_v31 l_v32 l_v33 p_h p_v1 p_v2
+
+# Fields to output
+sampling.fields = velocity temperature
+
+# Definitions for each probe
+sampling/l_v11.type = LineSampler
+sampling/l_v11.num_points = 192
+sampling/l_v11.start = 500.0 500.0 5.208333333333333
+sampling/l_v11.end = 500.0 500.0 1994.7916666666667
```

Comment:
```suggestion
sampling/l_v11.start = 500.0 500.0 0.0
sampling/l_v11.end = 500.0 500.0 2000.0
```

Reply:
I was planning to probe only at the cell centers. Will the previous one be ok then?